app/controllers/consultations_controller.rb

```diff
@@ -1,18 +0,0 @@
-class ConsultationsController < DocumentsController
-  def index
-    filter_params = params.except(:controller, :action, :format, :_)
```

Comment:
Isn't this still needed? I think it's responsible for this redirect: https://www.gov.uk/government/consultations